tests/kw_db_test.sh

```diff
@@ -119,15 +119,21 @@ function test_execute_command_db()
 
   output=$(execute_command_db 'SELECT * FROM not_a_table;' 2>&1)
   ret="$?"
-  expected='Error: no such table: not_a_table'
+  expected='no such table: not_a_table'
   assert_equals_helper 'Invalid table.' "$LINENO" "$ret" 1
-  assert_equals_helper 'Wrong output.' "$LINENO" "$output" "$expected"
+  if ! grep -qi "$expected" <<< "$output"; then
+    fail "line ($LINENO): Wrong output."
+    return
+  fi
```

Comment:
This is something interesting that can be useful in other places; how about creating an assert helper for this? Maybe we can have something like `assert_substring_match` in the `tests/utils.sh`.